### PR TITLE
Update sl-jdbc.md

### PR DESCRIPTION
**Pull request by rpourzand**
We're making some changes to our API (not released yet), so I'm preemptively adding info.

## Checklist

- [ ] Review the [Content style guide](https://github.com/dbt-labs/docs.getdbt.com/blob/current/contributing/content-style-guide.md) so my content adheres to these guidelines.
- [ ] For [docs versioning](https://github.com/dbt-labs/docs.getdbt.com/blob/current/contributing/single-sourcing-content.md#about-versioning), review how to [version a whole page](https://github.com/dbt-labs/docs.getdbt.com/blob/current/contributing/single-sourcing-content.md#adding-a-new-version) and [version a block of content](https://github.com/dbt-labs/docs.getdbt.com/blob/current/contributing/single-sourcing-content.md#versioning-blocks-of-content).
- [ ] Add a checklist item for anything that needs to happen before this PR is merged, such as "needs technical review" or "change base branch."

Adding new pages (delete if not applicable):
- [ ] Add page to `website/sidebars.js`
- [ ] Provide a unique filename for the new page

Removing or renaming existing pages (delete if not applicable):
- [ ] Remove page from `website/sidebars.js`
- [ ] Add an entry `website/static/_redirects`
- [ ] Run link testing locally with `npm run build` to update the links that point to the deleted page

### CODE REVIEW

**Review by DevonFulcher — APPROVED** [review]
Looks good, based on Paul's demo. I'm not sure when these features go live though.

File: website/docs/docs/dbt-cloud-apis/sl-graphql.md [hunk 1]
```diff
@@ -432,25 +432,83 @@ mutation {
...
+
+```
+
+Assuming the user is querying metric_0 and metric_1 together, a valid filter would be:
+
+  * `"{{ TimeDimension('metric_time', 'year') }} > '2020-01-01'"`
+
+Invalid Filters would be:
+
+  `* "{{ TimeDimension('metric_time') }} > '2020-01-01'"` - metrics in the query
```
Comment (plypaul) [inline website/docs/docs/dbt-cloud-apis/sl-graphql.md]: The asterisk for the bullet point should be outside of the quote.

File: website/docs/docs/dbt-cloud-apis/sl-jdbc.md [hunk 2]
```diff
@@ -254,11 +254,70 @@ Where filters in API allow for a filter list or string. We recommend using the f
...
 Where Filters have a few objects that you can use:
 
-- `Dimension()` - Used for any categorical or time dimensions. If used for a time dimension, granularity is required -  `Dimension('metric_time').grain('week')` or `Dimension('customer__country')`
+- `Dimension()` - Used for any categorical or time dimensions. `Dimension('metric_time').grain('week')` or `Dimension('customer__country')`
 
 - `Entity()` - Used for entities like primary and foreign keys - `Entity('order_id')`
 
-Note: If you prefer a more explicit path to create the `where` clause, you can optionally use the `TimeDimension` feature. This helps separate out categorical dimensions from time-related ones. The `TimeDimesion` input takes the time dimension name and also requires granularity, like this: `TimeDimension('metric_time', 'MONTH')`.
+
+Note: If you prefer a more explicit path to create the `where` clause, you can optionally use the `TimeDimension` feature. This helps separate out categorical dimensions from time-related ones. The `TimeDimension` input takes the time dimension name optionally requires granularity, like this: `TimeDimension('metric_time', 'month')`.
```
Comment (plypaul) [inline website/docs/docs/dbt-cloud-apis/sl-jdbc.md]: Maybe `explicit path` -> `strongly typed` since that phrase was used before?

**Review by plypaul — APPROVED** [review]
LGTM aside form comments

**Comment by plypaul** [comment]
The ambiguous resolution stack has not been released yet, so do we need to wait for that to go out before merging this?

File: website/docs/docs/dbt-cloud-apis/sl-graphql.md (hunk 1 above)
Comment (rpourzand) [reply website/docs/docs/dbt-cloud-apis/sl-graphql.md]: good find!

File: website/docs/docs/dbt-cloud-apis/sl-jdbc.md [hunk 3]
```diff
@@ -254,11 +254,70 @@ Where filters in API allow for a filter list or string. We recommend using the f
...
+
+For both `TimeDimension()` and `Dimension()` objects, the grain is only required if in the WHERE filter if the aggregation time dimensions for the measures & metrics associated with the where filter have different grains.
```
Comment (Jstein77) [inline website/docs/docs/dbt-cloud-apis/sl-jdbc.md]: Grain is still required when querying a time dimension using `Dimension()`. For example you would still need to add `Dimensions('metric_time','day')`.

File: website/docs/docs/dbt-cloud-apis/sl-graphql.md [hunk 4]
```diff
@@ -428,29 +430,91 @@ mutation {
...
     where:[{sql: "{{ Dimension('customer__customer_type') }} = 'new'"}, {sql:"{{ Dimension('metric_time').grain('month') }} > '2022-10-01'"}]
     ) {
      queryId
     }
 }
 ```
 
+For both `TimeDimension()` and `Dimension()` objects, the grain is only required in the WHERE filter if the aggregation time dimensions for the measures & metrics associated with the where filter have different grains. 
+
+For example, consider this Semantic Model and Metric Configuration which contains two metrics that are aggregated across different time grains. This example shows a single Semantic Model, but the same goes for metrics across more than one Semantic Model.
```
Comment (mirnawong1) [inline website/docs/docs/dbt-cloud-apis/sl-graphql.md]:
```suggestion
For example, consider this Semantic model and Metric configuration, which contains two metrics that are aggregated across different time grains. This example shows a single semantic model, but the same goes for metrics across more than one semantic model.
```

File: website/docs/docs/dbt-cloud-apis/sl-jdbc.md [hunk 5]
```diff
@@ -254,11 +254,70 @@ Where filters in API allow for a filter list or string. We recommend using the f
 
 Where Filters have a few objects that you can use:
 
-- `Dimension()` - Used for any categorical or time dimensions. If used for a time dimension, granularity is required -  `Dimension('metric_time').grain('week')` or `Dimension('customer__country')`
+- `Dimension()` - Used for any categorical or time dimensions. `Dimension('metric_time').grain('week')` or `Dimension('customer__country')`
```
Comment (mirnawong1) [inline website/docs/docs/dbt-cloud-apis/sl-jdbc.md]:
```suggestion
- `Dimension()` &mdash; Used for any categorical or time dimensions. `Dimension('metric_time').grain('week')` or `Dimension('customer__country')`
```

File: website/docs/docs/dbt-cloud-apis/sl-jdbc.md [hunk 6]
```diff
@@ -254,11 +254,70 @@ Where filters in API allow for a filter list or string. We recommend using the f
 
 Where Filters have a few objects that you can use:
 
-- `Dimension()` - Used for any categorical or time dimensions. If used for a time dimension, granularity is required -  `Dimension('metric_time').grain('week')` or `Dimension('customer__country')`
+- `Dimension()` &mdash; Used for any categorical or time dimensions. `Dimension('metric_time').grain('week')` or `Dimension('customer__country')`
+
+- `TimeDimension()` - Used as a more explicit definition for time dimensions, optionally takes in a granularity `TimeDimension('metric_time', 'month')`
```
Comment (mirnawong1) [inline website/docs/docs/dbt-cloud-apis/sl-jdbc.md]:
```suggestion
- `TimeDimension()` &mdash;  Used as a more explicit definition for time dimensions, optionally takes in a granularity `TimeDimension('metric_time', 'month')`
```

File: website/docs/docs/dbt-cloud-apis/sl-jdbc.md [hunk 7]
```diff
@@ -254,11 +254,70 @@ Where filters in API allow for a filter list or string. We recommend using the f
 
 Where Filters have a few objects that you can use:
 
-- `Dimension()` - Used for any categorical or time dimensions. If used for a time dimension, granularity is required -  `Dimension('metric_time').grain('week')` or `Dimension('customer__country')`
+- `Dimension()` &mdash; Used for any categorical or time dimensions. `Dimension('metric_time').grain('week')` or `Dimension('customer__country')`
+
+- `TimeDimension()` &mdash;  Used as a more explicit definition for time dimensions, optionally takes in a granularity `TimeDimension('metric_time', 'month')`
 
 - `Entity()` - Used for entities like primary and foreign keys - `Entity('order_id')`
```
Comment (mirnawong1) [inline website/docs/docs/dbt-cloud-apis/sl-jdbc.md]:
```suggestion
- `Entity()` &mdash;  Used for entities like primary and foreign keys - `Entity('order_id')`
```

File: website/docs/docs/dbt-cloud-apis/sl-jdbc.md [hunk 8]
```diff
@@ -254,11 +254,70 @@ Where filters in API allow for a filter list or string. We recommend using the f
...
-- `Dimension()` - Used for any categorical or time dimensions. If used for a time dimension, granularity is required -  `Dimension('metric_time').grain('week')` or `Dimension('customer__country')`
+- `Dimension()` &mdash; Used for any categorical or time dimensions. `Dimension('metric_time').grain('week')` or `Dimension('customer__country')`
 
-- `Entity()` - Used for entities like primary and foreign keys - `Entity('order_id')`
+- `TimeDimension()` &mdash;  Used as a more explicit definition for time dimensions, optionally takes in a granularity `TimeDimension('metric_time', 'month')`
+
+- `Entity()` &mdash;  Used for entities like primary and foreign keys - `Entity('order_id')`
+
+
+For `TimeDimension()`, the grain is only required in the WHERE filter if the aggregation time dimensions for the measures & metrics associated with the where filter have different grains.
```
Comment (mirnawong1) [inline website/docs/docs/dbt-cloud-apis/sl-jdbc.md]:
```suggestion
For `TimeDimension()`, the grain is only required in the `WHERE` filter if the aggregation time dimensions for the measures and metrics associated with the where filter have different grains. 
```

File: website/docs/docs/dbt-cloud-apis/sl-jdbc.md [hunk 9]
```diff
@@ -254,11 +254,70 @@ Where filters in API allow for a filter list or string. We recommend using the f
...
 
-- `Entity()` - Used for entities like primary and foreign keys - `Entity('order_id')`
+- `TimeDimension()` &mdash;  Used as a more explicit definition for time dimensions, optionally takes in a granularity `TimeDimension('metric_time', 'month')`
+
+- `Entity()` &mdash;  Used for entities like primary and foreign keys - `Entity('order_id')`
+
+
+For `TimeDimension()`, the grain is only required in the `WHERE` filter if the aggregation time dimensions for the measures and metrics associated with the where filter have different grains. 
+
+For example, consider this Semantic Model and Metric Config which contains two metrics that are aggregated across different time grains. This example shows a single Semantic Model, but the same goes for metrics across more than one Semantic Model.
```
Comment (mirnawong1) [inline website/docs/docs/dbt-cloud-apis/sl-jdbc.md]:
```suggestion
For example, consider this Semantic model and Metric config, which contains two metrics that are aggregated across different time grains. This example shows a single semantic model, but the same goes for metrics across more than one semantic model.
```

File: website/docs/docs/dbt-cloud-apis/sl-jdbc.md [hunk 10]
```diff
@@ -254,11 +252,62 @@ Where filters in API allow for a filter list or string. We recommend using the f
 
 Where Filters have a few objects that you can use:
 
-- `Dimension()` - Used for any categorical or time dimensions. If used for a time dimension, granularity is required -  `Dimension('metric_time').grain('week')` or `Dimension('customer__country')`
+- `Dimension()` &mdash; Used for any categorical or time dimensions. `Dimension('metric_time').grain('week')` or `Dimension('customer__country')`
```
Comment (matthewshaver) [inline website/docs/docs/dbt-cloud-apis/sl-jdbc.md]:
```suggestion
- `Dimension()` &mdash; Used for any categorical or time dimensions. `Dimension('metric_time').grain('week')` or `Dimension('customer__country')`.
```

File: website/docs/docs/dbt-cloud-apis/sl-jdbc.md [hunk 11]
```diff
@@ -254,11 +252,62 @@ Where filters in API allow for a filter list or string. We recommend using the f
...
+
+- `TimeDimension()` &mdash;  Used as a more explicit definition for time dimensions, optionally takes in a granularity `TimeDimension('metric_time', 'month')`
```
Comment (matthewshaver) [inline website/docs/docs/dbt-cloud-apis/sl-jdbc.md]:
```suggestion
- `TimeDimension()` &mdash;  Used as a more explicit definition for time dimensions, optionally takes in a granularity `TimeDimension('metric_time', 'month')`.
```

File: website/docs/docs/dbt-cloud-apis/sl-jdbc.md [hunk 12]
```diff
@@ -254,11 +252,62 @@ Where filters in API allow for a filter list or string. We recommend using the f
...
+
+- `Entity()` &mdash;  Used for entities like primary and foreign keys - `Entity('order_id')`
```
Comment (matthewshaver) [inline website/docs/docs/dbt-cloud-apis/sl-jdbc.md]:
```suggestion
- `Entity()` &mdash;  Used for entities like primary and foreign keys - `Entity('order_id')`.
```

**Review by Jstein77 — APPROVED** [review]
This looks right to me. I think we need to add some more examples of how to query multi-hop dimensions in the CLI - but I can add that in a different PR.

**Comment by rpourzand** [comment]
@mirnawong1  this is ready to merge!

File: website/docs/docs/dbt-cloud-apis/sl-jdbc.md [hunk 13]
```diff
@@ -326,8 +376,9 @@ semantic_layer.query(metrics=['food_order_amount', 'order_gross_profit'],
   limit=10,
   order_by=[-'order_gross_profit']
   }}
-``` 
-If you are ordering by an object that's been operated on (e.g., change granularity), or you are using the full object notation, descending order must look like:
+```
+
+If you are ordering by an object that's been operated on (e.g., you changed the the granularity of the time dimension), or you are using the full object notation, descending order must look like:
```
Comment (mirnawong1) [inline website/docs/docs/dbt-cloud-apis/sl-jdbc.md]:
```suggestion
If you are ordering by an object that's been operated on (for example, you changed the the granularity of the time dimension), or you are using the full object notation, descending order must look like:
```

File: website/docs/docs/dbt-cloud-apis/sl-jdbc.md [hunk 14]
```diff
@@ -266,11 +264,62 @@ Where filters in API allow for a filter list or string. We recommend using the f
...
+
+```
+
+Assuming the user is querying `metric_0` and `metric_1` together in a single request, a valid `WHERE` filter would be:
 
-- `Entity()` - Used for entities like primary and foreign keys - `Entity('order_id')`
+  * `"{{ TimeDimension('metric_time', 'year') }} > '2020-01-01'"`
 
-Note: If you prefer a more explicit path to create the `where` clause, you can optionally use the `TimeDimension` feature. This helps separate out categorical dimensions from time-related ones. The `TimeDimesion` input takes the time dimension name and also requires granularity, like this: `TimeDimension('metric_time', 'MONTH')`.
+Invalid Filters would be:
```
Comment (mirnawong1) [inline website/docs/docs/dbt-cloud-apis/sl-jdbc.md]:
```suggestion
Invalid filters would be:
```

File: website/docs/docs/dbt-cloud-apis/sl-jdbc.md [hunk 15]
```diff
@@ -364,16 +415,26 @@ semantic_layer.query(metrics=['food_order_amount', 'order_gross_profit'],
 
 ## FAQs
 
-<FAQ path="Troubleshooting/sl-alpn-error" />
+<FAQ path="Troubleshooting/sl-alpn-error" /><br />
```
Comment (mirnawong1) [inline website/docs/docs/dbt-cloud-apis/sl-jdbc.md]:
```suggestion
<FAQ path="Troubleshooting/sl-alpn-error" />
```

File: website/docs/docs/dbt-cloud-apis/sl-graphql.md [hunk 16]
```diff
@@ -443,29 +445,91 @@ mutation {
...
-    groupBy:[{name: "customer__customer_type"}, {name: "metric_time", grain: MONTH}]
+    groupBy:[{name: "customer__customer_type"}, {name: "metric_time", grain: month}]
     where:[{sql: "{{ Dimension('customer__customer_type') }} = 'new'"}, {sql:"{{ Dimension('metric_time').grain('month') }} > '2022-10-01'"}]
     ) {
      queryId
     }
 }
 ```
 
+For both `TimeDimension()`, the grain is only required in the WHERE filter if the aggregation time dimensions for the measures & metrics associated with the where filter have different grains.
```
Comment (mirnawong1) [inline website/docs/docs/dbt-cloud-apis/sl-graphql.md]:
```suggestion
For both `TimeDimension()`, the grain is only required in the WHERE filter if the aggregation time dimensions for the measures and metrics associated with the where filter have different grains. 
```

**Review by mirnawong1 — APPROVED** [review]

**Comment by mirnawong1** [comment]
double checked and looking great! merging 🚢 - thank you so much @rpourzand !

**Pull request merged**